master/pkg/tasks/task_gc.go

```diff
@@ -61,12 +65,15 @@ func (g GCCkptSpec) ToTaskSpec() TaskSpec {
 					0o600,
 					tar.TypeReg,
 				),
+				// TODO: Do we use this file? I don't think so???
+				// If we do we need to update add glob
 				g.Base.AgentUserGroup.OwnedArchiveItem(
 					"checkpoint_gc/checkpoints_to_delete.json",
-					[]byte(jsonify(g.ToDelete)),
```

Comment:
do we still use these files? As far as I can tell we pass everything through command line arguments now?

Reply:
you do need this file, because otherwise there are system limit on how many or how long the cli args can be.

If we're relying on cli args at the moment, then it's a regression.  I originally added this file to get around that limitation.

Reply:
Checkpoint storage is still transferred as a file, I'm not sure if it was determined that checkpoint storage is the argument at risk of hitting the system limit or if it was a regression.

This PR awhile back which added delete checkpoints switched passing checkpoint uuids as a list instead of a file and never deleted the file. 
https://github.com/determined-ai/determined/pull/4246/files#diff-ebcdb45e3a87d3548c878e57fab45c41041815fec850f14fe74a755b1b2bc7bdR74

I'll just switch back to json file for uuids.